RSSReader-Maruyama/Modules/Login/LoginViewController.swift

```diff
@@ -0,0 +1,126 @@
+//
+//  LoginViewController.swift
+//  RSSReader-Maruyama
+//
+//  Created by 丸山翔 on 2020/09/03.
+//  Copyright © 2020 丸山翔. All rights reserved.
+//
+
+import UIKit
+import AuthenticationServices
+import FBSDKCoreKit
+import FBSDKLoginKit
+import GoogleSignIn
+
+class LoginViewController: UIViewController {
+    
+    @IBOutlet weak var loginButtonStackView: UIStackView!
+    
+    private lazy var appleLoginButton: ASAuthorizationAppleIDButton = {
+        let appleLoginButton = ASAuthorizationAppleIDButton()
+        appleLoginButton.addTarget(self, action: #selector(didPushLoginButtonApple), for: .touchUpInside)
+        appleLoginButton.heightAnchor.constraint(equalToConstant: 45).isActive = true
+        
+        return appleLoginButton
+    }()
+    
+    private let SELECT_FEEDS_SEGUE_ID = "selectFeeds"
+    
+    private let userRepository = UserRepository()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        loginButtonStackView.addArrangedSubview(appleLoginButton)
+        GIDSignIn.sharedInstance()?.delegate = self
+        GIDSignIn.sharedInstance()?.presentingViewController = self
+    }
+    
+    private func loginSuccessAction(userID: String) {
```

Comment:
```suggestion
    private func didLoginSuccess(userID: String) {
```
〜が完了した、〜された系のメソッドはdid~~に統一するようにしましょう。